apps/expo-go/android/expoview/src/main/java/versioned/host/exp/exponent/ExponentDevBundleDownloadListener.kt

```diff
@@ -18,7 +18,9 @@ class ExponentDevBundleDownloadListener(private val listener: DevBundleDownloadP
     listener.onProgress(status, done, total)
   }
 
-  override fun onFailure(cause: Exception) {
-    listener.onFailure(cause)
+  override fun onFailure(cause: Exception?) {
+    if(cause !== null){
+      listener.onFailure(cause)
+    }
```

Comment:
```suggestion
    cause?.let {
      listener.onFailure(it)
    }
```

nit: maybe more kotlin style